vite.config.ts

```diff
@@ -18,13 +18,23 @@ export default defineConfig(({ mode }) => {
 
   return {
     build: {
-      chunkSizeWarningLimit: 900,
       outDir,
       rollupOptions: {
         input: {
-          main: resolve(__dirname, 'index.html'),
+          index: resolve(__dirname, 'index.html'),
```

Comment:
If I'm not mistaken, this only changes the name, but not _merge_ the two files:

```
dist/assets/index-BZn5sF-c.js                                          6.56 kB │ gzip:   2.74 kB
dist/assets/editingConfigs-WubQ1JNa.js                                26.47 kB │ gzip:   8.25 kB
dist/assets/ScrivitoBootstrapIconEditor-DTXMz-zT.js                   47.51 kB │ gzip:  15.12 kB
dist/assets/index-DOwQ58pJ.js                                        107.26 kB │ gzip:  30.25 kB
```

Compare to one previous commit:

```
dist/assets/main-Bfrvei51.js                                           6.56 kB │ gzip:   2.74 kB
dist/assets/editingConfigs-WubQ1JNa.js                                26.47 kB │ gzip:   8.25 kB
dist/assets/ScrivitoBootstrapIconEditor-DTXMz-zT.js                   47.51 kB │ gzip:  15.12 kB
dist/assets/index-DOwQ58pJ.js                                        107.26 kB │ gzip:  30.25 kB
```